core/src/main/scala/org/apache/spark/util/Utils.scala

```diff
@@ -1945,44 +1945,144 @@ private[spark] object Utils extends Logging {
    * @return (service: T, port: Int)
    */
   def startServiceOnPort[T](
-      startPort: Int,
-      startService: Int => (T, Int),
-      conf: SparkConf,
-      serviceName: String = ""): (T, Int) = {
+                             startPort: Int,
+                             startService: Int => (T, Int),
+                             conf: SparkConf,
+                             serviceName: String = ""): (T, Int) = {
```

Comment:
You should indent it back to 4 spaces, as it was in the original code (same style guide applies)